src/components/TopLevelModals/index.tsx

```diff
@@ -17,13 +19,17 @@ export default function TopLevelModals() {
 
   const blockedAccountModalOpen = useModalIsOpen(ApplicationModal.BLOCKED_ACCOUNT)
   const { account } = useWeb3React()
+  const location = useLocation()
 
   useAccountRiskCheck(account)
   const open = Boolean(blockedAccountModalOpen && account)
   return (
     <>
       <AddressClaimModal isOpen={addressClaimOpen} onDismiss={addressClaimToggle} />
       <ConnectedAccountBlocked account={account} isOpen={open} />
+      {location.pathname.includes('/swap') && <NftExploreBanner />}
+      {location.pathname.includes('/tokens') && <NftExploreBanner />}
+      {location.pathname.includes('/pool') && <NftExploreBanner />}
       <Bag />
       {useNftFlag() === NftVariant.Enabled && <TransactionCompleteModal />}
       {useNftFlag() === NftVariant.Enabled && <AirdropModal />}
```

Comment:
BTW, I know this is completely unrelated to this PR, but can we rewrite this too? Why not a single:
```
const isNftEnabled = useNftFlag()

{isNftEnabled && (
  <>
    <TransactionCompleteModal />
    <AirdropModal />
  </>
)}
```